server/controllers/tripController.js

```diff
@@ -11,15 +11,19 @@ class TripController {
    * This method sets a trip type.
    * @param {object} trip The user's request.
    * @param {string} returnDate The response.
+   * @param {integer} cityNumber The response.
    * @returns {object} The status and some data of the trip.
    */
-  static async setTripType(trip, returnDate) {
+  static async setTripType(trip, returnDate, cityNumber) {
     try {
-      if (returnDate) {
+      if (returnDate && cityNumber === 1) {
```

Comment:
Why do you have to check whether a city's number is equal to 1?

Reply:
to specify that the user has requested one destination, which will tell us it is not a multi-city trip...just a return trip